src/cryptography/hazmat/bindings/openssl/osrandom_engine.py

```diff
@@ -4,14 +4,12 @@
 
 from __future__ import absolute_import, division, print_function
 
-INCLUDES = """
-#ifdef _WIN32
-#include <Wincrypt.h>
-#else
-#include <fcntl.h>
-#include <unistd.h>
-#endif
-"""
+import os
+
+with open(os.path.join(
+    os.path.dirname(__file__), "src/osrandom_engine.h")
+) as f:
```

Comment:
Slight nitpick, but can we change how this is written so that the parens are aligned:

```
with open(os.path.join(
    os.path.dirname(__file__), "src/osrandom_engine.h"
)) as f:
```